cypcore/Ledger/Graph.cs

```diff
@@ -150,6 +151,9 @@ private IDisposable BlockmaniaListener()
                     {
                         var blockGraphs = _unitOfWork.BlockGraphRepository
                             .WhereAsync(x => new ValueTask<bool>(x.Block.Round == NextRound())).AsTask().Result;
+
+                        if (blockGraphs.Count < 2) return;
```

Comment:
Please explain the purpose of this.

Reply:
I sure its self explanatory what the short circuit statement is doing?